packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.jsx

```diff
@@ -1,54 +1,36 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import IconAlert from 'terra-icon/lib/icon/IconAlert';
 
-const icon = <IconAlert width="4em" height="4em" />;
-// Block style is required since images are inline by default. This could be resolved by someones normalize.css
-const image = <img style={{ display: 'block' }} height="75" width="75" src="https://github.com/cerner/terra-core/raw/master/terra.png" alt="a happy panda" />;
 const ipsum = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.';
 const simpleText = <div>{ipsum}</div>;
-const textWithPadding = <div style={{ paddingLeft: '10px', paddingRight: '10px', border: '1px solid red' }}>{ipsum}</div>;
-
-const ArrangeWrapper = props => (
-  <div style={{ width: '500px', border: '1px solid black' }}>
-    {props.children}
-  </div>
+const textWithBlueBorder = <div style={{ border: '2px solid #0079BE' }}>{ipsum}</div>;
+
+const alignExampleDiv = (
+  <div
+    style={{
+      backgroundColor: '#000',
+      display: 'block',
+      height: '20px',
+      margin: '1px',
```

Comment:
nit: I'd drop the margin here, I don't want people to think that the arrange is padding components within it.

Reply:
Updated: [68c3a20](https://github.com/cerner/terra-core/pull/2353/commits/68c3a206ca8b941ea1e9f9e7a3e6fad41525c455).